angular-ngc/tools/BUILD.bazel

```diff
@@ -30,3 +31,10 @@ ts_project(
         "//:node_modules/@types/node",
     ],
 )
+
+exports_files(
```

Comment:
I don't think you have to do this when it's only used internally in this folder

Reply:
I have to add `exports_files` since in `karma.bzl` this file is loaded by `//tools:karma.conf.js`.